addons/l10n_ec/__manifest__.py

```diff
@@ -17,6 +17,9 @@
     'depends': [
         'account',
         'base_iban',
+        'l10n_latam_base',
+        'account_debit_note',
```

Comment:
As we added dependency of l10n_latam_invoice_document to account_debit_note, we can remove account_debit_note here.  @nla-odoo